Form/Transformer/EntityToIdObjectTransformer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Form\Transformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Doctrine\Common\Persistence\ObjectManager;
+
+/**
+ * Class EntityToIdObjectTransformer
+ *
+ * @author Marc Juchli <mail@marcjuch.li>
+ */
+class EntityToIdObjectTransformer implements DataTransformerInterface {
+
+    /**
+     * @var ObjectManager
+     */
+    private $om;
+
+    /**
+     * @var String
+     */
+    private $entityName;
+
+    /**
+     * @param ObjectManager $om
+     */
+    public function __construct(ObjectManager $om, $entityName)
+    {
+        $this->entityName = $entityName;
+        $this->om = $om;
+    }
+
+    /**
+     * Do nothing.
+     *
+     * @param  Object|null $object
+     * @return Object
+     */
+    public function transform($object)
+    {
+        if (null === $object) {
+            return "";
+        }
+
+        return $object->getId();
```

Comment:
Does this make sense? Shouldn't it be `return array($object->getId())`? Does it make sense to check if the object actually has a `getId()` method first?